internal/component/database_observability/postgres/component.go

```diff
@@ -297,8 +298,27 @@ func (c *Component) startCollectors() error {
 		level.Error(c.opts.Logger).Log("msg", "failed to start ConnectionInfo collector", "err", err)
 		return err
 	}
+
 	c.collectors = append(c.collectors, ciCollector)
 
+	if collectors[collector.SchemaTableName] {
+		stCollector, err := collector.NewSchemaTable(collector.SchemaTableArguments{
+			DB:              dbConnection,
+			InstanceKey:     c.instanceKey,
+			CollectInterval: c.args.CollectInterval,
```

Comment:
I can set a configurable collect interval as 1 min might be too frequent before caching. What do y'all think?

Reply:
We could define the collect interval at the collector level, set it to something higher than 1m, and put a TODO to revisit later if/when we add caching.